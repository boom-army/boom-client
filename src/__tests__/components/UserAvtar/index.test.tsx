import React from "react";
import PersonAvatar from "../../../components/UserAvatar/index";
import renderer from "react-test-renderer";

const avatar: string = `https://sosol-prod.s3.us-west-2.amazonaws.com/images/Screenshot%20from%202022-02-01%2018-34-57.png`;

describe("Avtar", () => {
  test("renders correctly when User have no avatar", () => {
    const tree = renderer
      .create(<PersonAvatar avatar={undefined && ""} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  test("renders correctly when User have avatar", () => {
    const tree = renderer.create(<PersonAvatar avatar={avatar} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
