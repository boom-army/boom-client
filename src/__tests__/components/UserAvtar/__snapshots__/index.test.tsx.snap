// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Avtar renders correctly when User have avatar 1`] = `
<div
  className="MuiAvatar-root MuiAvatar-circular avatar css-1wlk0hk-MuiAvatar-root"
>
  <img
    className="MuiAvatar-img css-1pqm26d-MuiAvatar-img"
    src="https://sosol-prod.s3.us-west-2.amazonaws.com/images/Screenshot%20from%202022-02-01%2018-34-57.png"
  />
</div>
`;

exports[`Avtar renders correctly when User have no avatar 1`] = `
<div
  className="MuiAvatar-root MuiAvatar-circular MuiAvatar-colorDefault avatar css-2s90m6-MuiAvatar-root"
>
  <svg
    aria-hidden={true}
    className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
    data-testid="PersonIcon"
    focusable="false"
    viewBox="0 0 24 24"
  >
    <path
      d="M12 12c2.21 0 4-1.79 4-4s-1.79-4-4-4-4 1.79-4 4 1.79 4 4 4zm0 2c-2.67 0-8 1.34-8 4v2h16v-2c0-2.66-5.33-4-8-4z"
    />
  </svg>
</div>
`;
